system_core/datadog_checks/system_core/system_core.py

```diff
@@ -35,7 +35,6 @@ def check(self, instance):
         cpu_freq = psutil.cpu_freq(percpu=True)
         self.log.debug('CPU frequency: %s', str(cpu_freq))
         for i, cpu in enumerate(cpu_freq):
-            if Platform.is_unix():
-                # Per-cpu frequency retrieval (Linux only)
-                tags = instance_tags + ['core:{0}'.format(i)]
-            self.gauge('system.core.frequency', cpu._asdict().get('current'), tags=tags)
+            # Per-cpu frequency retrieval (Unix only)
+            tags = instance_tags + ['core:{0}'.format(i)] if Platform.is_unix() else instance_tags
```

Comment:
nit - at first, I understood the comment as "we retrieve the `system.core.frequency` metric only on Unix" which is not the case, it's that we do tag it with the cpu only on Unix. Am I just dumb or could we rephrase it a bit?

Reply:
nah, it's not a very clear comment. I just took it over from the previous implementation, I've improved it now. Thanks for the catch!